decidim-admin/app/views/decidim/admin/scopes/search_results.json.erb

```diff
@@ -0,0 +1,9 @@
+<%
```

Comment:
What about doing this on the controller layer (maybe via a presenter or serializer) instead? JSON is not markup language and doing this here looks a bit out of place (or maybe I'm missing something?)